local_adaptation/README.md

```diff
@@ -0,0 +1,28 @@
+##  Local Adaptation NWF
+
+This is a simulation in which local adaptation is achieved through QTL mutations arising in a 20kb region of the 1e6bp genomic element. Neutral mutations may be overlayed for downstream analysis (say, using msprime), as they are excluded from this example. 
+
+`LAimg_greyscale.png` is the example greyscale image used in the current script, but this can be easily exchanged for another greyscale image. If you choose to upload an RBG image, make sure to edit `mapImage.floatK` in line 61 to `mapImage.floatG`. Greyscale is recommended. 
+
+An interesting aspect to this script is how we approach fitness scaling. Notice that we first measure local density, like in other examples, but then we also construct phenotypes and fitness effects from QTLs. The phenotypes of individuals are calculated as the sum of the `m2` mutations (QTL). So then, we have these two elements of fitness scaling: 
+
+first in line 76:
+```
+inds.fitnessScaling = 1 / (1 + RHO * competition);
+```
+
+second in line 87:
+```
+inds.fitnessScaling =   inds.fitnessScaling * qtl_fit;
+```
+where qtl_fit is defined as:
+```
+qtl_fit = 1 + dnorm(phenotype, optimum, SK);
```

Comment:
Seems the rescaling makes the `1 +` bit unnecessary?

Reply:
See my comment elsewhere.  The 1+ is a value for a, whereas the rescaling is a value for b.  They actually have quite different effects.  There's a whole family of possible fitness functions, for different values of a and b.  But of course if you want a to be 0, and that works for you, that's fine; then you can leave the 1+ out.  :->